python/py-python-coveralls/Portfile

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-python-coveralls
+version             2.9.3
+revision            0
+categories-append   devel
+
+license             Apache-2
+maintainers         nomaintainer
+platforms           darwin
+
+description         Python API for http://coveralls.io.
+
+long_description    ${description}.
+
+homepage            https://github.com/z4r/python-coveralls
+
+distname            ${python.rootname}-${version}
+
+checksums           rmd160  a6b60bc37aa14b2ba1bae0e56daa5e07f5a4fa3a \
+                    sha256  bfaf7811e7dc5628e83b6b162962a4e2485dbff184b30e49f380374ed1bcee55 \
+                    size    9036
+
+supported_archs     noarch
```

Comment:
Nitpick. Please move this line to just below the `platforms` line like most other Portfiles.

Reply:
Done thanks.